classes/migration/install/SubmissionFilesMigration.php

```diff
@@ -42,27 +42,28 @@ public function up(): void
             $table->index(['submission_id'], 'submission_files_submission_id');
             //  pkp/pkp-lib#5804
             $table->index(['file_stage', 'assoc_type', 'assoc_id'], 'submission_files_stage_assoc');
-            $table->foreign('file_id')->references('file_id')->on('files');
+            $table->foreign('file_id')->references('file_id')->on('files')->onDelete('cascade');;
         });
 
         // Article supplementary file metadata.
         Schema::create('submission_file_settings', function (Blueprint $table) {
-            $table->bigInteger('submission_file_id');
+            $table->bigInteger('submission_file_id')->nullable(false)->unsigned();
```

Comment:
Are these necessary? Just curious; I think `NOT NULL` is a default, and `unsigned` is accurate, but we haven't generally specified it on IDs.

Reply:
Looks like this was because the submission_files table uses bigIncrements, which is an unsigned type. We should be using [foreignId](https://laravel.com/docs/9.x/migrations#column-method-foreignId) columns to refer to these.

Reply:
Change that to `$table->foreignId('submission_file_id');`